### PR TITLE
bar airbnb

### PR DESCRIPTION
**Pull request by ZoyaCanepa**
- [DEMO LINK](https://baranovskazoya.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://baranovskazoya.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,5 +12,43 @@
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+
+    <form
+    class="form"
+    action="/"
+    method="get"
+    data-qa="big"
+  >
```
Comment (OleksiiBlezniuk) [inline src/index.html:21]:
```suggestion
    <form
      class="form"
      action="/"
      method="get"
      data-qa="big"
    >
```

Comment (OleksiiBlezniuk) [reply src/index.html:21]: Fix indentations everywhere please

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding-top: 12px;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:21]: Do you really need this padding?

File: src/index.html [hunk 3]
```diff
@@ -12,5 +12,43 @@
   </head>
   <body>
     <h1>Search bar airbnb</h1>
```
Comment (OleksiiBlezniuk) [inline src/index.html:14]: Please do not ignore tests, remove that title

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+.form {
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.search {
+  font-family: "Avenir", sans-serif;
+  display: block;
```
Comment (OleksiiBlezniuk) [inline src/style.css:30]: Inputs are inline-blocks by default, why do you need this?

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  padding-top: 12px;
+}
+
+.form {
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.search {
+  font-family: "Avenir", sans-serif;
```
Comment (OleksiiBlezniuk) [inline src/style.css:29]: Usually we set font related styles to `html` tag, then almost every element will inherit it (if not, you can set `inherit` yourself)

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+.search:focus {
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+}
+
+.search--small::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
```
Comment (OleksiiBlezniuk) [inline src/style.css:70]: This styles should be set for input text, not for placeholder, it may break your tests

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Please don't ignore tests, I believe that you can fix them (at least most of them)

File: src/index.html [hunk 7]
```diff
@@ -11,6 +11,43 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    class="form__big"
+    action="#"
+    method="post"
+    data-qa="big"
+  >
```
Comment (OleksiiBlezniuk) [inline src/index.html:20]:
```suggestion
    <form
      class="form__big"
      action="#"
      method="post"
      data-qa="big"
    >
```

Comment (OleksiiBlezniuk) [reply src/index.html:20]: I've written about it last during last review

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:20]: Have you used this class?

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+  font-size: 16px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  font-weight: bold;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:52]: You forgot to disable outline

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
There is a bug 🙂 Fix it please, hint in comments

Screenshot 2022-08-17 at 17 12 39

File: src/index.html (hunk 7 above)
Comment (baranovskazoya) [reply src/index.html]: Sorry, I misunderstood

File: src/index.html [hunk 10]
```diff
@@ -11,6 +11,43 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    class="form__big"
+    action="#"
+    method="post"
+    data-qa="big"
+    >
```
Comment (OleksiiBlezniuk) [inline src/index.html:20]:
```suggestion
    <form
      class="form__big"
      action="#"
      method="post"
      data-qa="big"
    >
```

File: src/index.html [hunk 11]
```diff
@@ -11,6 +11,43 @@
...
+      <input
+      class="search"
+      name="searchBig"
+      type="text"
+      data-qa="keypress"
+      placeholder="Try &quot;Los Angeles&quot;"
+    >
```
Comment (OleksiiBlezniuk) [inline src/index.html:27]:
```suggestion
      <input
        class="search"
        name="searchBig"
        type="text"
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      >
```

File: src/index.html [hunk 12]
```diff
@@ -11,6 +11,43 @@
...
+      <img
+      class="icon"
+      src="images/Search.svg"
+      alt="search"
+    >
```
Comment (OleksiiBlezniuk) [inline src/index.html:32]:
```suggestion
      <img
        class="icon"
        src="images/Search.svg"
        alt="search"
      >
```

File: src/index.html [hunk 13]
```diff
@@ -11,6 +11,43 @@
...
+    </form>
+    <form
+    class="form__small"
+    action="#"
+    method="post"
+    data-qa="small"
+    >
```
Comment (OleksiiBlezniuk) [inline src/index.html:39]:
```suggestion
    <form
      class="form__small"
      action="#"
      method="post"
      data-qa="small"
    >
```

File: src/index.html [hunk 14]
```diff
@@ -11,6 +11,43 @@
...
+      <input
+      class="search search--small"
+      name="searchSmall"
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot;"
+    >
```
Comment (OleksiiBlezniuk) [inline src/index.html:45]:
```suggestion
      <input
        class="search search--small"
        name="searchSmall"
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
      >
```

File: src/index.html [hunk 15]
```diff
@@ -11,6 +11,43 @@
...
+      <img
+      class="icon icon--small"
+      src="images/Search.svg"
+      alt="search"
+    >
```
Comment (OleksiiBlezniuk) [inline src/index.html:50]:
```suggestion
      <img
        class="icon icon--small"
        src="images/Search.svg"
        alt="search"
      >
```

File: src/index.html [hunk 16]
```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    class="form__big"
```
Comment (OleksiiBlezniuk) [inline src/index.html:16]: Have you used this class?

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Bug is still here, and you remove right styles for form 🥲
Screenshot 2022-08-17 at 19 17 52
Apply them to both forms

File: src/style.css [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+
+.form__big {
+  margin-top: 20px;
+  position: relative;
+}
+
+.form__small {
+  margin-top: 20px;
+  position: relative;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:25]:
```suggestion
.form {
  margin-top: 20px;
  position: relative;
}
```

Comment (OleksiiBlezniuk) [reply src/style.css:25]: No need to duplicate styles, just create one class and apply it to every element that should have this styles. Don't repeat yourself

**Review by OleksiiBlezniuk — APPROVED** [review]
GJ, but next time try to avoid styles duplicate, don't forget that you can create one class and apply this class to every element that should have these styles. Class is like a template, that can be reused

**Pull request closed without merging**